### PR TITLE
HERITAGE-72 Update results list card template to meet OHOS requirements

### PR DESCRIPTION
**Pull request by garethmead**
Ticket URL: [HERITAGE-72](https://national-archives.atlassian.net/browse/HERITAGE-72)

## About these changes

Update search results cards to match OHOS requirements. Hard coded elements included with comments until API data is available

## How to check these changes

Search results can be found at - `/search/catalogue/?group=digitised`

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes
- [x] Ensured that PR includes only commits relevant to the ticket
- [x] Waited for all CI jobs to pass before requesting a review
- [ ] Added/updated tests and documentation where relevant

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

[HERITAGE-72]: https://national-archives.atlassian.net/browse/HERITAGE-72?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by garethmead** [comment]
The icon used differs slightly from the one in the design, as the font-awesome style used for that has been removed from tna-frontend here - https://github.com/nationalarchives/tna-frontend/commit/d512254b49c2bab3fb7c25609e0a44920a9d436e. As the style isn’t used anywhere else it would be a bit of overhead for the sake of a single icon

**Review by TNA-Allan — CHANGES_REQUESTED** [review]
I've put a comment in the ticket
Re: 
> I’d recommend not to remove, 

> Icons for ‘Held by’, ‘Date’ and ‘Reference’

> Rows for ‘Held by’ and ‘Reference’

> as this belongs to TNA (2nd) or NON TNA (3rd) buckets. It will not show for DCC (1st bucket) when we integrate the data for DCC.

> but will show for TNA and NON TNA

**Comment by TNA-Allan** [comment]
> Leave this as is as Digitised is not in the scope as per the Wireframe. It will be removed by HERITAGE-92: Remove unrequired ETNA code - IN PROGRESS The bucket itself will not show.
> 
> (updated)

Hi @garethmead 
As per latest comments, it needs to be removed from TNA as well. So retain removal of condition with `record.is_digitised` 
Thanks :)

**Comment by garethmead** [comment]
> > Leave this as is as Digitised is not in the scope as per the Wireframe. It will be removed by HERITAGE-92: Remove unrequired ETNA code - IN PROGRESS The bucket itself will not show.
> > (updated)
> 
> Hi @garethmead As per latest comments, it needs to be removed from TNA as well. So retain removal of condition with `record.is_digitised` Thanks :)

I've reinstated the additional fields but kept them commented out. With regard to the icons, as they're not in the design I've kept them removed, as we'd also need to find an icon for `Collection`

**Review by ahosgood — APPROVED** [review]

File: templates/search/blocks/search_results__list-card--grid.html [hunk 1]
```diff
@@ -30,88 +30,26 @@ <h4 class="tna-heading-s">
...
             {{ record.listing_description }}
         </p>
-        <dl class="tna-dl tna-dl--plain tna-dl--icon-padding tna-!--margin-bottom-s">
-            {% if record.held_by %}
-                <dt>
-                    <i class="fa-solid fa-landmark"></i>
-                    Held by
-                </dt>
-                <dd>{{ record.held_by }}</dd>
-            {% endif %}
```
Comment (TNA-Allan) [inline templates/search/blocks/search_results__list-card--grid.html:45]: same, please reinstate this as commented inside new dl, for the purpose of tna,non-tna buckets

File: templates/search/blocks/search_results__list-card--grid.html [hunk 2]
```diff
@@ -30,88 +30,26 @@ <h4 class="tna-heading-s">
...
-
+        <dl class="tna-dl tna-dl--plain tna-!--margin-bottom-s">
             {% if record.date_created %}
-                <dt>
-                    <i class="fa-solid fa-calendar"></i>
-                    Date
-                </dt>
```
Comment (TNA-Allan) [inline templates/search/blocks/search_results__list-card--grid.html:51]: same, please reinstate this as commented inside new dl, for the purpose of tna,non-tna buckets

File: templates/search/blocks/search_results__list-card--grid.html [hunk 3]
```diff
@@ -30,88 +30,26 @@ <h4 class="tna-heading-s">
...
-                IAID:
-            </dt>
-
-            <dd>{{ record.iaid }}</dd>>
-            <dt>
-                Score:
-            </dt>
-            <dd;>{{ record.score }}</dd>
-            -->
-            {% comment %}End debug attributes.{% endcomment %}
```
Comment (TNA-Allan) [inline templates/search/blocks/search_results__list-card--grid.html:96]: same, please reinstate this as commented inside new dl, for the purpose of tna,non-tna buckets

File: templates/search/blocks/search_results__list-card.html [hunk 4]
```diff
@@ -11,92 +11,64 @@
...
         </h3>
 
         <p class="search-results__list-card-description">
             {{ record.listing_description }}
         </p>
 
-        <dl class="tna-dl tna-dl--plain tna-dl--icon-padding tna-!--margin-bottom-s">
-            {% if record.held_by %}
-            <dt>
-                <i class="fa-solid fa-landmark"></i>
```
Comment (TNA-Allan) [inline templates/search/blocks/search_results__list-card.html:31]: same, please reinstate this as commented inside new dl, for the purpose of tna,non-tna buckets

File: templates/search/blocks/search_results__list-card.html [hunk 5]
```diff
@@ -11,92 +11,64 @@
...
+                This will require some additional logic once we are working with OHOS data
+                {% if record.held_by %}
+                    <dt>Held by</dt>
+                    <dd>{{ record.held_by }}</dd>
+                {% endif %}
+            {% endcomment %}
 
             {% if record.date_created %}
-            <dt>
-                <i class="fa-solid fa-calendar"></i>
```
Comment (TNA-Allan) [inline templates/search/blocks/search_results__list-card.html:41]: same, please reinstate this as commented inside new dl, for the purpose of tna,non-tna buckets

File: templates/search/blocks/search_results__list-card.html [hunk 6]
```diff
@@ -11,92 +11,64 @@
...
-                <date>{{ record.date_created }}</date></dd>
+                <dt>Date</dt>
+                <dd>
+                    <date>{{ record.date_created }}</date>
+                </dd>
             {% endif %}
 
-            {% if record.reference_number %}
-            <dt>
-                <i class="fa-solid fa-database"></i>
```
Comment (TNA-Allan) [inline templates/search/blocks/search_results__list-card.html:50]: same, please reinstate this as commented inside new dl, for the purpose of tna,non-tna buckets

**Review by TNA-Allan — CHANGES_REQUESTED** [review]

**Comment by TNA-Allan** [comment]
> > > Leave this as is as Digitised is not in the scope as per the Wireframe. It will be removed by HERITAGE-92: Remove unrequired ETNA code - IN PROGRESS The bucket itself will not show.
> > > (updated)
> > 
> > 
> > Hi @garethmead As per latest comments, it needs to be removed from TNA as well. So retain removal of condition with `record.is_digitised` Thanks :)
> 
> I've reinstated the additional fields but kept them commented out. With regard to the icons, as they're not in the design I've kept them removed, as we'd also need to find an icon for `Collection`

Thanks @garethmead , with regard to the etna icons, I've suggested to reinstate as commented as you've done with others. When we have TNA,NON-TNA data, will take a look and uncomment accordingly.

**Comment by garethmead** [comment]
@TNA-Allan Icons added back in with the commented fields. For simplicity I've kept all of these within the same `dl` and we can figure out what needs to stay/be removed once the data is integrated

**Review by TNA-Allan — APPROVED** [review]
Investigated and noted this ticket succeeds HERITAGE-89 Remove grid and list options from sort by area on Search Results
Looks good. Thanks @garethmead

**Pull request merged**